Client.php

```diff
@@ -105,8 +126,6 @@ class Client extends Connection
      */
     protected $_stack = [];
 
-
```

Comment:
Keep these lines please. We always have 3 empty lines between constants/attributes and methods.